tests/unit/test_document.py

```diff
@@ -522,7 +522,11 @@ def test_convert_document_to_annotate_file_response():
 
     actual = doc.convert_document_to_annotate_file_response()
 
-    assert actual != AnnotateFileResponse()
+    assert actual.responses[0].text_annotations[0].description == "Invoice"
+    assert len(actual.responses[0].text_annotations) == 86
```

Comment:
Do we have more detailed tests? such as comparing the responses?

Reply:
Added a comparison for the full `AnnotateFileResponse`